### PR TITLE
Created pagewide card

### PR DESCRIPTION
**Pull request by pallavikadam-srijan**
Created page-wide featured content card

### CODE REVIEW

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@import "../../../../assets/scss/variables";
+@import "../../../../assets/scss/mixins";
+@import "../../../../assets/scss/base";
+
+.wide-card {
+	&-container {
+		border-top: 2px solid #000000;
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:7]: @pallavikadam-srijan Please use proper variables

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		padding-top: 16px;
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:8]: @pallavikadam-srijan Please use proper variables

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+					display: block;
+					position: absolute;
+					transition: opacity .5s ease;
+					background-image: linear-gradient(
+				45deg,#ffeb00,transparent 80%);
+				}
+			}
+		}
+	}
+}
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:78]: @@pallavikadam-srijan Please use proper spacing variables and mixins

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+				&::before {
+					top: 0;
+					left: 0;
+					opacity: 0;
+					content: "";
+					width: 100%;
+					height: 100%;
+					display: block;
+					position: absolute;
+					transition: opacity .5s ease;
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:71]: @pallavikadam-srijan please use the mixin

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comment on PR

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/PageWideCard.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import { P } from '../../../../Atom/Base-typography/Paragraph/Paragraph';
+import { CtaButton } from '../../../../Atom/Buttons-and-labels/Cta_button/CtaButton';
+import BackgroundImg from '../../../../assets/images/Pagewide.jpg';
+
+export const PageWideCard = ({ label, title, paragraph, button }) => {
+  return (
+    <div className="wide-card-container">{label}
+      <div className="grid-x wide-card-container__small-size">
+        <div className="cell medium-6 wide-card__wrapper">
+          <span className="wide-card__wrapper--summary">
```
[the remark this answers is not in the document]
Comment (pallavikadam-srijan) [reply stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/PageWideCard.jsx:13]: done

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/PageWideCard.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+            <a className="wide-card__wrapper--summary-link" href="#1">
+              <Heading type="4" className="wide-card__wrapper--summary-link-title" label={title} />
+              <P className="wide-card__wrapper--summary-link-paragraph" label={paragraph} />
+            </a>
```
[the remark this answers is not in the document]
Comment (pallavikadam-srijan) [reply stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/PageWideCard.jsx:17]: done

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss (hunk 1 above)
Comment (pallavikadam-srijan) [reply stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:7]: done

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss (hunk 2 above)
Comment (pallavikadam-srijan) [reply stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:8]: done

File: stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss [hunk 7]
```diff
@@ -0,0 +1,102 @@
+@import '../../../../assets/scss/variables';
+@import '../../../../assets/scss/mixins';
+@import '../../../../assets/scss/base';
+
+.wide-card {
+  &-container {
+    border-top: 2px solid #000;
```
Comment (surbhi90) [inline stories/Organism/Blocks/FeaturedContentCard/PagewideContentCard/pagewidecard.scss:7]: @pallavikadam-srijan Please use variables

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comment

**Review by surbhi90 — APPROVED** [review]

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**